src/olympia/lib/akismet/tests.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from django.conf import settings
+
+import mock
+
+from olympia.amo.tests import addon_factory, TestCase, user_factory
+from olympia.ratings.models import Rating
+
+from .models import AkismetReport
+
+
+class TestAkismetReportsModel(TestCase):
+
+    def setUp(self):
+        patcher = mock.patch('olympia.lib.akismet.models.requests.post')
```

Comment:
Personally, I'd prefer using `responses` compared to patching which mocks on socket-level compared to mocking specific APIs.

See `TestWebextUpdateDescriptions` for a very good example.

Reply:
According to blame I wrote that test and now can't remember it, heh.

Reply:
https://github.com/mozilla/addons-server/pull/9127/commits/7bbf9d8547127f7de106cfdb80aee28d9a5166e6